[2025-05-18T08:07:59.217+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_price_pipeline.validate_data manual__2025-05-18T08:07:48.235490+00:00 [queued]>
[2025-05-18T08:07:59.229+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_price_pipeline.validate_data manual__2025-05-18T08:07:48.235490+00:00 [queued]>
[2025-05-18T08:07:59.230+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-18T08:07:59.247+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): validate_data> on 2025-05-18 08:07:48.235490+00:00
[2025-05-18T08:07:59.252+0000] {standard_task_runner.py:60} INFO - Started process 639 to run task
[2025-05-18T08:07:59.256+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_price_pipeline', 'validate_data', 'manual__2025-05-18T08:07:48.235490+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/flight_price_.py', '--cfg-path', '/tmp/tmptepudnjk']
[2025-05-18T08:07:59.258+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask validate_data
[2025-05-18T08:07:59.311+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_price_pipeline.validate_data manual__2025-05-18T08:07:48.235490+00:00 [running]> on host d05befceb79d
[2025-05-18T08:07:59.466+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='flight_price_pipeline' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-18T08:07:48.235490+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-18T08:07:48.235490+00:00'
[2025-05-18T08:08:00.265+0000] {data_validation.py:23} INFO - Connected to MySQL for validation
[2025-05-18T08:08:00.987+0000] {data_validation.py:27} INFO - Loaded 57000 rows for validation
[2025-05-18T08:08:01.031+0000] {data_validation.py:42} INFO - Validation passed successfully
[2025-05-18T08:08:01.094+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-18T08:08:01.138+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_price_pipeline, task_id=validate_data, execution_date=20250518T080748, start_date=20250518T080759, end_date=20250518T080801
[2025-05-18T08:08:01.241+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-18T08:08:01.287+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
